### DISCUSSION TITLE
Calculator app

### DISCUSSION THREAD

**Discussion by hexisXz** [InfiniTimeOrg/InfiniTime#1831] [Ideas]
I think there should be a really simple calculator just for convenience. Idk if I am dumb or blind but I am pretty sure there is no calculator on the watch so it would be incredibly useful if there was like a simple calculator app on the watch.

**Comment by FintasticMan** [comment]
There is a PR for that: #1483.

**Reply by hexisXz** [reply]
is there a way to install it?

**Reply by hexisXz** [reply]
nvm i figured it out now i just need to make a bug report because it wont build
